extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -182,12 +208,6 @@ describe('Project: sqlproj content operations', function (): void {
 		ssdtUri = project.getSystemDacpacSsdtUri(constants.msdbDacpac);
 		should.equal(uri.fsPath, Uri.parse(path.join('$(NETCoreTargetsPath)', 'SystemDacpacs', '130', constants.msdbDacpac)).fsPath);
 		should.equal(ssdtUri.fsPath, Uri.parse(path.join('$(DacPacRootPath)', 'Extensions', 'Microsoft', 'SQLDB', 'Extensions', 'SqlServer', '130', 'SqlSchemas', constants.msdbDacpac)).fsPath);
-
-		project.changeTargetPlatform(constants.targetPlatformToVersion.get(constants.sqlAzure)!);
```

Comment:
removing this since a reference to MSDB can't be added when the project targets Azure